### ISSUE TITLE
Check for reassignment error

### ISSUE THREAD

**Issue by windard** [kisielk/errcheck#217]
In the following code, no linter error is raised. Is this case catchable?

```
func doo() (int, int, error) {
    var (
        x, y int
        err  error
    )
    x, err = foo()
    y, err = a(x)
    return x, y, err
}

func do() (x int, y int, err error) {
    x, err = foo()
    y, err = a(x)
    return x, y, err
}
```

**Comment by dtcaciuc** [comment]
Not at the moment (see #7). We currently recommend to use https://staticcheck.io (in particular https://staticcheck.io/docs/checks/#SA4006)
